interop-tests/src/test/scala/org/apache/pekko/grpc/scaladsl/PowerApiSpec.scala

```diff
@@ -182,6 +182,37 @@ abstract class PowerApiSpec(backend: String)
 
       metadataServer.terminate(3.seconds)
     }
+
+    "pre-announce trailers in the headers" in {
```

Comment:
Please include a comment here as well explaining why it is important to pre-announce trailers: link to the RFC (though with the caveat that it's strictly speaking about chunked and not about h2) and mentioning the tyk bug. Maybe add an issue to https://github.com/TykTechnologies/tyk so you can link to it?

Reply:
Yeah I will need some help from my SRE colleagues to open an issue in there. They are on vacation now, but it would be good to come up with a fix for it as well

Reply:
sure I can move it 👀